RELEASE_NOTES.md

```diff
@@ -0,0 +1,74 @@
+# Release v0.11.0
+
+## New Features
+
+* Support team wise NFS storage, including:
+    - An NFS configuration plug-in and a commandline tool. [#2346](https://github.com/Microsoft/pai/pull/2346)
+    - A simple NFS-job submit plug-in. [#2358](https://github.com/Microsoft/pai/pull/2358)
+    
```

Comment:
Refer to [Simplified Job Submission for OpenPAI + NFS deployment](https://github.com/Microsoft/pai/wiki/Simplified-Job-Submission-for-OpenPAI-with-NFS-deployment) for more details.